core/src/mindustry/entities/comp/StatusComp.java

```diff
@@ -135,6 +135,7 @@ public void update(){
                 damageMultiplier *= entry.effect.damageMultiplier;
                 reloadMultiplier *= entry.effect.reloadMultiplier;
                 buildSpeedMultiplier *= entry.effect.buildSpeedMultiplier;
+                dragMultiplier *= entry.effect.dragMultiplier;
```

Comment:
This needs to be reset to 1 every update above the loop.

Reply:
right. I forgot